src/utiltest.cpp

```diff
@@ -46,6 +46,12 @@ CWalletTx GetValidReceive(ZCJoinSplit& params,
                           inputs, outputs, 2*value, 0, false};
     mtx.vjoinsplit.push_back(jsdesc);
 
+    // Shielded Output
+    OutputDescription od;
+    uint256 cm;
+    od.cm = cm;
```

Comment:
This does nothing: both `od.cm` and `cm` will initialize to the all-zeroes byte array.